### PR TITLE
Fix swagger of FunctionBase display problem

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <ranxiaolong716@gmail.com>

### Motivation

Fix swagger of FunctionBase display problem and use `Pulsar Function` instead of `function` or `functions` .

Before fix:

image

After fix:

image

### CODE REVIEW

**Comment by wolfstudy** [comment]
@sijie @jerrypeng @Jennifer88huang PTAL thanks.

**Comment by Jennifer88huang** [comment]
@Anonymitaet Could you please help review? Thank you.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java [hunk 1]
```diff
@@ -70,90 +70,90 @@ public WorkerService get() {
...
                             "  already exists on worker host)] from which worker can download the package.  \n" +
-                            "py  \n" +
-                            "  Path to the main Python file/Python Wheel file for the function (if the function is written in Python).  \n" +
-                            "go  \n" +
-                            "  Path to the main Go executable binary for the function (if the function is written in Go).  \n" +
-                            "logTopic  \n" +
-                            "  The topic to which the functions logs are produced.  \n" +
-                            "maxMessageRetries  \n" +
+                            "- **py**  \n" +
+                            "  Path to the main Python file/Python Wheel file for the Pulsar Function (if the Pulsar Function is written in Python).  \n" +
```
Comment (Anonymitaet) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java]:
```suggestion
                            "  Path to the main Python file or Python wheel file for the Pulsar Function (if the Pulsar Function is written in Python).  \n" +
```

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java [hunk 2]
```diff
@@ -70,90 +70,90 @@ public WorkerService get() {
...
-                            "secrets  \n" +
-                            "  This is a map of secretName(that is how the secret is going to be accessed in the function via context) to an object that" +
+                            "- **secrets**  \n" +
+                            "  This is a map of secretName(that is how the secret is going to be accessed in the Pulsar Function via context) to an object that" +
                             "  encapsulates how the secret is fetched by the underlying secrets provider. The type of an value here can be found by the" +
                             "  SecretProviderConfigurator.getSecretObjectType() method. \n" +
-                            "cleanupSubscription  \n" +
-                            "  Whether the subscriptions the functions created/used should be deleted when the functions are deleted.  \n",
+                            "- **cleanupSubscription**  \n" +
+                            "  Whether the subscriptions of a Pulsar Function created/used should be deleted when the Pulsar Function is deleted.  \n",
```
Comment (Anonymitaet) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java]:
```suggestion
                            "  Whether the subscriptions of a Pulsar Function created or used should be deleted when the Pulsar Function is deleted.  \n",
```

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java [hunk 3]
```diff
@@ -178,89 +178,89 @@ public void registerFunction(
...
                             "  already exists on worker host)] from which worker can download the package.  \n" +
-                            "py  \n" +
-                            "  Path to the main Python file/Python Wheel file for the function (if the function is written in Python).  \n" +
-                            "go  \n" +
-                            "  Path to the main Go executable binary for the function (if the function is written in Go).  \n" +
-                            "logTopic  \n" +
-                            "  The topic to which the functions logs are produced.  \n" +
-                            "maxMessageRetries  \n" +
+                            "- **py**  \n" +
+                            "  Path to the main Python file/Python Wheel file for the Pulsar Function (if the Pulsar Function is written in Python).  \n" +
```
Comment (Anonymitaet) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java]:
```suggestion
                            "  Path to the main Python file or Python wheel file for the Pulsar Function (if the Pulsar Function is written in Python).  \n" +
```

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java [hunk 4]
```diff
@@ -511,121 +511,121 @@ public void putFunctionState(final @PathParam("tenant") String tenant,
...
             final @PathParam("namespace") String namespace,
-            @ApiParam(value = "The name of functions")
+            @ApiParam(value = "The name of a Pulsar Function")
             final @PathParam("functionName") String functionName) {
         functions.stopFunctionInstances(tenant, namespace, functionName, clientAppId(), clientAuthData());
     }
 
     @POST
-    @ApiOperation(value = "Start function instance", response = Void.class)
+    @ApiOperation(value = "Start instance of a Pulsar Function", response = Void.class)
```
Comment (Anonymitaet) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java]:
```suggestion
    @ApiOperation(value = "Start an instance of a Pulsar Function", response = Void.class)
```

**Comment by Anonymitaet** [comment]
@Jennifer88huang glad to help

**Comment by wolfstudy** [comment]
> Can you also fix sources and sinks in a separated pull request? I believe they also have the display problem.

Sure, will fix them.

**Comment by wolfstudy** [comment]
run java8 tests

**Comment by wolfstudy** [comment]
```
2019-08-07\T\07:59:50.178 [ERROR] Failures: 
2019-08-07\T\07:59:50.179 [ERROR] org.apache.pulsar.functions.worker.PulsarFunctionStateTest.testPulsarFunctionState(org.apache.pulsar.functions.worker.PulsarFunctionStateTest)
2019-08-07\T\07:59:50.179 [INFO]   Run 1: PASS
2019-08-07\T\07:59:50.179 [ERROR]   Run 2: PulsarFunctionStateTest.testPulsarFunctionState:341 expected [5] but found [3]
```

run java8 tests

**Comment by wolfstudy** [comment]
run java8 tests

```
2019-08-07\T\09:45:15.433 [ERROR] Failures: 
2019-08-07\T\09:45:15.434 [ERROR]   PulsarFunctionStateTest.setup » StatusRuntime INTERNAL: http2 exception
```

**Comment by wolfstudy** [comment]
run java8 tests

**Comment by wolfstudy** [comment]
```
2019-08-07\T\16:00:37.320 [ERROR] Tests run: 30, Failures: 2, Errors: 0, Skipped: 20, Time elapsed: 79.929 s <<< FAILURE! - in org.apache.pulsar.client.impl.BrokerClientIntegrationTest
2019-08-07\T\16:00:37.321 [ERROR] testCloseConnectionOnBrokerRejectedRequest(org.apache.pulsar.client.impl.BrokerClientIntegrationTest)  Time elapsed: 2.309 s  <<< FAILURE!
org.apache.pulsar.client.api.PulsarClientException$TooManyRequestsException: Failed due to too many pending lookup requests
	at org.apache.pulsar.client.api.PulsarClientException.unwrap(PulsarClientException.java:255)
	at org.apache.pulsar.client.impl.ProducerBuilderImpl.create(ProducerBuilderImpl.java:88)
	at org.apache.pulsar.client.impl.BrokerClientIntegrationTest.testCloseConnectionOnBrokerRejectedRequest(BrokerClientIntegrationTest.java:535)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

run java8 tests

**Comment by wolfstudy** [comment]
run java8 tests

**Comment by sijie** [comment]
// org.apache.pulsar.functions.worker.PulsarFunctionStateTest.testPulsarFunctionState #4907

run java8 tests

**Pull request merged**